scvi/model/_linear_scvi.py

```diff
@@ -125,3 +127,49 @@ def get_loadings(self) -> pd.DataFrame:
         )
 
         return loadings
+
+    @staticmethod
+    def setup_anndata(
+        adata: AnnData,
+        batch_key: Optional[str] = None,
```

Comment:
I believe this one needs the labels key as well as it has a dispersion option of gene-label